chromium_presubmit_config.json5

```diff
@@ -153,7 +153,8 @@
       "test/data/speedreader/",
     ],
     "CheckStableMojomChanges": [
-      "components/brave_rewards/common/mojom/rewards_engine\\.mojom",
+      "components/brave_rewards/common/mojom/.*\\.mojom",
+      "components/services/bat_rewards/public/interfaces/rewards_engine_factory\\.mojom",
```

Comment:
why does this need to be added?

Reply:
This presubmit checks tries to find mojo interfaces that have (accidentally) changed. When mojom files are moved, the check frequently throws an error because it can't find the "old" file. After this PR is merged, we can remove these lines.